bag/datasets/bag/batch.py

```diff
@@ -317,6 +317,15 @@ def process(self):
     def process_row(self, r):
         pk = r['identificatie']
         gemeente_id = self.gemeentes.get(r['ligtIn:BRK.GME.identificatie'])
+        wkt_geometrie = r['geometrie']
+        if wkt_geometrie:
+            geometrie = geo.get_multipoly(wkt_geometrie)
+            if not geometrie:
+                log.error(f"Woonplaats {pk} has no valid geometry; skipping")
+                return None
+        else:
+            log.warning(f"Ligplaats {pk} has no geometry")
```

Comment:
Waar komt dat verschil in Woonplaats en Ligplaats vandaan?

Reply:
Copy paste probleem. Goed opgemerkt :)